fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -692,7 +716,12 @@ private void handleQueryStmt(List<PlanFragment> fragments, List<ScanNode> scanNo
         // 2. If this is a query, send the result expr fields first, and send result data back to client.
         RowBatch batch;
         MysqlChannel channel = context.getMysqlChannel();
-        boolean isOutfileQuery = queryStmt.hasOutFileClause();
+        boolean isOutfileQuery;
+        if (queryStmt instanceof QueryStmt) {
```

Comment:
The `QueryStmt` and `QueryStatement` are very confusing.

Reply:
You are right, it does cause confusion. However, considering that QueryStmt will soon be completely replaced, this name will no longer appear, and this process will be very short, and it can be deleted after Query-related operations（insert, create view) are fully supported. On the other hand it was hard to find a new name to express what QueryStatement means, so I used QueryStatement to represent the new parser result